docs/custom-flows/email-sms-otp.mdx

```diff
@@ -1,85 +1,67 @@
+### Enable SMS OTP
 
-### Enable passwordless authentication using the Clerk API
+To use SMS OTP as an authentication strategy, you first need to enable it in the Clerk Dashboard.
 
-In case one of the above integration methods doesn't cover your needs, you can make use of lower-level commands and create a completely custom passwordless authentication flow.
+1. Navigate to the Clerk Dashboard.
+2. Go to **User & Authentication > [Email, Phone, and Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)** in the sidebar menu.
+3. In the **Contact information** section, ensure *only* **Phone number** is enabled. This will allow users to add phone numbers to their accounts. If you would like to use email OTP instead, you can enable the **Email address** option instead of the phone number option.
+4. In the **Authentication strategies** section of this page, ensure *only* **SMS verification code** is enabled. Ensure **Password** is toggled off, as you are priotirizing passwordless, SMS OTP-only authentication in this guide. If you would like to use email OTP instead, you can enable the **Email verification code** strategy instead of the SMS strategy.
 
-<Callout type="warning">
-  You still need to configure your instance in order to enable passwordless authentication, as described at the top of this guide.
-</Callout>
-
-#### Sign up using a custom flow (Clerk API)
+### Create the OTP sign-up flow
 
-The passwordless sign-up flow is a process that requires users to provide their authentication identifier (email address or phone number) and a one-time code that is sent to them. The important thing to note here is that a user's email address or phone number needs to be verified before the registration is completed.
+To sign up a user using an OTP, you need to:
 
-A successful sign-up consists of the following steps:
+1. Initiate the sign-up process by collecting the user's identifier, which for this example is a phone number.
+2. Prepare the verification, which sends a one-time code to the given identifier.
+3. Attempt to complete the verification by supplying the one-time code.
+4. If the verification is successful, complete the sign-up process by creating the user account and setting their session as active.
 
-- Initiate the sign-up process by collecting the user's identifier (email address or phone number).
-- Prepare the identifier verification.
-- Attempt to complete the identifier verification.
+In the following example, the user is asked to provide their phone number. Once the user submits their phone number, a one-time code is sent to them. The user is then prompted to enter the code to complete the sign-up process. If the verification is successful, the user's session is set as active. To ease the development process, Clerk-related error messages will be displayed on the user interface.
```

Comment:
```suggestion
In the following example:
1. The user is asked to provide their phone number.
1. Once the user submits their phone number, a one-time code is sent to them.
1. The user is then prompted to enter the code to complete the sign-up process.
1. If the verification is successful, the user's session is set as active.

To ease the development process, Clerk-related error messages will be displayed on the user interface.
```
I suggest we break this up so it's easier to read.

Reply:
I didn't see any errors in the UI. Do you mean in the console?

Reply:
oh it doesn't exist in the nextjs examples - I can add it

Reply:
addressed - [7f8ea58](https://github.com/clerk/clerk-docs/pull/981/commits/7f8ea58f22ed67fc3a10cfdbf0ee0319df81d035)